### ISSUE TITLE
Mutations of parent classes not recognized in DevTools

### ISSUE THREAD

**Issue by econic** [ktsn/sinai#15]
Mutations are not recognized in the dev tools if the mutation method is located in a parent class and called on a sub class.

This is my current implementation:
```
abstract class AbstractMutations extends Mutations {
  myMutation() { ... }
}
```
And this is the mutations class that the mutation gets called on:
```
class ConcreteMutations extends AbstractMutations { }
```

The app is working as expected, it's just not debuggable like that.

**Comment by econic** [comment]
The above mutation is inside an abstract class. That's the only special thing i can think about.

**Comment by ktsn** [comment]
@econic Thanks for reporting this! This has been fixed in v0.1.6.

**Comment by econic** [comment]
Fix confirmed, thx!
